.gitignore

```diff
@@ -43,3 +43,13 @@ gen-external-apklibs
 # Gradle
 .gradle
 build
+ParseUI-Widget/._src
```

Comment:
The drive I have my projects saved in, is an exFat file system, which automatically creates these files.  I just auto-ignore them, as they keep popping up.